docker/grafana/setup-grafana.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/sh
+
+# Remove the whitelist from the config file
+sed -i '/^whitelist/d' /etc/grafana/grafana.ini \
```

Comment:
Now that we are using the existing grafana config it introduces a whitelist, which is not needed in docker (see https://grafana.com/docs/grafana/latest/auth/auth-proxy/#full-walk-through-using-docker). This whitelist prevents grafana from loading and needs to be removed. Unfortunately, while environment variables allow you to override entries for configs, they do not allow you to set a blank value / remove a config entry. Therefore, I needed a way to remove the whitelist entry from the config that was copied over to /etc/grafana. This `sed` entry does just that and allows grafana to load while also enabling the auth proxy.